README.md

```diff
@@ -12,19 +12,98 @@ other components of the system, including the Istio mixer and the Istio proxy me
 If you have a question about the Istio Manager or have a problem using it, please
 [file an issue](https://github.com/istio/manager/issues/new).
 
-## Build instructions ##
+## Setting up a local dev environment ##
```

Comment:
Can we move this down and rename to Vagrant build method? Thanks for making sure Windows is supported with this approach. Currently, to build Manager you need Linux no matter what your favorite build environment is, so we should put that foremost.